tutorials/vpn-interop-guides/using-cloud-vpn-with-aws/index.md

```diff
@@ -0,0 +1,591 @@
+
+1. Configure the Amazon Virtual Private Gateway
+1. Configure the Amazon Customer Gateway
+1. Configure the Google Cloud Platform VPN
+1. Set up the VPN Connection
+1. Connect to GCP
+1. Test the tunnel
+
+## Getting Started
+
+The first step is to establish the base networking environment in AWS, which is called the Virtual Private Cloud (VPC).   Amazon provides [documentation](http://aws.amazon.com/documentation/vpc/) for getting started with AWS networking.  The basic concepts to understand are:
+
+-  **Virtual Private Cloud **– a customer-defined private network space in AWS.  
+-  **Virtual Private Gateway** – the VPN concentrator on the Amazon side of the VPN connection.
+-  **Customer Gateway** – an AWS reference to the remote IPsec endpoint.  In this case, the Google Cloud Platform VPN gateway.
+
+## IPsec Parameters 
+
+This table covers IPsec parameters to use when configuring VPN gateways and tunnels as described in this document._ _The IPsec connectivity covered in this guide uses the** pre-shared key** generated by AWS for authentication. AWS only supports IKEv1. Detailed GCP-supported IKEv1 ciphers can be found [here.](https://cloud.google.com/vpn/docs/concepts/advanced#supported_ike_ciphers)
+
+<table>
+<thead>
+<tr>
+<th><strong>Parameter</strong></th>
+<th><strong>Value</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>IPsec Mode</td>
+<td>ESP+Auth Tunnel mode (Site-to-Site)</td>
+</tr>
+<tr>
+<td>Authentication Protocol</td>
+<td>Pre-shared Key</td>
+</tr>
+<tr>
+<td>Key Exchange</td>
+<td>IKEv1</td>
+</tr>
+<tr>
+<td>Perfect Forward Secrecy (PFS)</td>
+<td>on</td>
+</tr>
+</tbody>
+</table>
+
+# IPsec VPN Using Static Routes
+
+## Reserve an external static IP address for GCP
+
+The AWS VPN configuration requires a remote VPN gateway IP address in advance. Reserve a static external IP address by selecting **External IP addresses** option under the [VPC networks menu](https://console.cloud.google.com/networking/addresses/list) option as shown below.
+
+![image](cloudvpnguide---zv350ow6efi.png)
+
+## Configuration - AWS
+
+For this exercise, create a new VPC network and subnet configuration using the AWS **VPC Wizard** to connect to Google Cloud Platform. The VPC Wizard steps through the creation and configuration of a new VPC network.
+
+### Creating the AWS VPC Network
+
+To get started, log in to the AWS Management Console and select **VPC** from the main services menu.  New AWS accounts all have a default VPC.  
+
+![image](cloudvpnguide---fsaahph6zzd.png)
+
+### 
+
+The first step is to select an IP subnet topology.  There are options for various combinations of private and public IP addressing, with or without VPN connectivity.  Once selected, this configuration cannot be changed.  For this test environment select **Private Subnet Only VPC with Hardware VPN Access**:
```

Comment:
In the screenshot, the option text reads "VPC with a Private Subnet Only and Hardware VPN Access". Not sure if the screenshot is wrong or the text is.